### ISSUE TITLE
net/http: something about 100-continue

### ISSUE THREAD

**Issue by gzw13999** [golang/go#9735]
If client http header contain  Expect: 100-continue

client post multipart/form-data Sometimes there will be
"multipart: Part Read: WSARecv tcp *.*.138.250:80: i/o timeout"  ParseMultipartForm(102400) -> /mime/multipart/multipart.go 239   
This is immediately returned error.
Did not arrive ReadTimeout

The same service, the same machine, open the IIS running do not have this problem.

**Comment by adg** [comment]
I'm having trouble understanding your problem. Can you rephrase?

**Comment by rsc** [comment]
Duplicate of #3665?
Brad, if so, and you don't want to fix this now, change to milestone Unplanned.

**Comment by anacrolix** [comment]
I think this is a duplicate. Any use of Expect: 100-continue in client code is currently not going to work the way expected and OP calls this out immediately.

**Comment by bradfitz** [comment]
Closing as duplicate of #3665
